vm-setup/roles/v1aX_integration_test/files/fetch_manifests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+mkdir -p /tmp/manifests
+
+manifests=(
+  bmh
+  cluster
+  deployment
+  machine
+  machinedeployment
+  machinehealthchecks
+  machinesets
+  machinepools
+  m3clustera
+  m3machine
+  metal3machinetemplate
+  kcp
+  kubeadmconfig
+  kubeadmconfigtemplates
+  kubeadmcontrolplane
+  replicaset
+)
+
+if [[ "${CAPI_VERSION}" == "v1alpha4" ]]; then 
+   manifests+=("m3ippool" "m3ipclaim" "ipaddress" "m3data" "m3dataclaim" "m3datatemplate") 
```

Comment:
now this is ippool, ipclaim and ipaddress. the rest is correct

Reply:
Fixed thanks